src/assets/sass/protocol/components/_footer.scss

```diff
@@ -70,7 +70,7 @@
     @include clearfix;
 
     @media #{$mq-md} {
-        border-bottom: 1px solid $color-gray-70;
+        border-bottom: 1px solid $color-gray-90;
```

Comment:
This new gray is very difficult to see on a black background. Should we either lighten it, or make it white?